default/scripting/ship_hulls/SH_COLONY_BASE.focs.txt

```diff
@@ -15,11 +16,23 @@ Hull
     buildTime = 3
     tags = [ "PEDIA_HULL_LINE_GENERIC" ]
     location = OwnedBy empire = Source.Owner
-    effectsgroups =
+    effectsgroups = [
         [[REGULAR_HULL_DETECTION]]
+
+        // There are weapons designed to kill a base hull with a single shot (Arc Disruptor)
+        // Also mines should be able to down base hull
+        // So we scale the base hull with the lower of those scaling factors
```

Comment:
why lower not higher?

Reply:
So that both (mines damage and arc disruptor single shot) are able to do the 1-shot kill.

Reply:
Another variant would be to not scale the base structure of the colony base hull at all. Reinforced hull scales with the weapon damage so that would always be correct (but that is probably a detail for followup PR when we actually set new default values)